sdk/tables/data-tables/src/utils/accountConnectionString.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { TableServiceClientOptions } from "..";
+import { ConnectionString } from "./connectionString";
```

Comment:
Any concerns that this module imports `connectionString` while the `connectionString` module imports this module?  I'm guessing not because ConnectionString is just a type here and so at runtime there is no circular reference?

I think I understand why the code is factored this way (logically this should be part of the connection string module, but the implementation of these two functions differs between browsers and node, so we pulled out only these methods into a new module who's implementation we could vary).

Reply:
Good catch, I'm creating an internalModels.ts to put this one and other models in the future